src/networkProviders/interface.ts

```diff
@@ -26,77 +32,100 @@ export interface INetworkProvider {
     getNetworkStatus(): Promise<NetworkStatus>;
 
     /**
-     * Fetches stake statistics.
+     * Fetches a block by nonce or by hash.
      */
-    getNetworkStakeStatistics(): Promise<NetworkStake>;
+    getBlock(blockArgs: GetBlockArguments): Promise<BlockOnNetwork>;
```

Comment:
I think this is removed from the interface in sdk-py, and implemented differently in API provider and proxy provider. How should we proceed?